src/controlCenter/widgets/volume/volume.vala

```diff
@@ -56,5 +129,35 @@ namespace SwayNotificationCenter.Widgets {
                 slider.set_value (device.volume);
             }
         }
+
+        private void active_sink_change (PulseSinkInput sink) {
+            foreach (var row in levels_listbox.get_children ()) {
+                if (row == null) continue;
+                var s = (SinkInputRow) row;
+                if (s.sink_input.cmp (sink)) {
+                    s.update (sink);
+                    break;
+                }
+            }
+        }
+
+        private void active_sink_added (PulseSinkInput sink) {
+            levels_listbox.add (new SinkInputRow (sink, client));
+            show_all ();
+        }
+
+        private void active_sink_removed (PulseSinkInput sink) {
+            foreach (var row in levels_listbox.get_children ()) {
+                if (row == null) continue;
+                var s = (SinkInputRow) row;
+                if (s.sink_input.cmp (sink)) {
+                    levels_listbox.remove (row);
+                    break;
+                }
+            }
+            if (levels_listbox.get_children ().length () == 0) {
+                print ("EMPTY\n");
+            }
+        }
```

Comment:
Seems like this one isn't working. None of the sink-inputs get removed when ex closing and opening Spotify.

![image](https://user-images.githubusercontent.com/35975961/228291007-ac989873-d63f-4db3-a96c-93ab1eaffc92.png)